app/src/main/java/de/westnordost/streetcomplete/quests/seating/AddSeating.kt

```diff
@@ -0,0 +1,61 @@
+package de.westnordost.streetcomplete.quests.seating
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement
+
+class AddSeating : OsmFilterQuestType<Seating>() {
+    override val elementFilter = """
+        nodes, ways with amenity ~ restaurant|cafe|fast_food|ice_cream|food_court|pub|bar
+        and !outdoor_seating
+    """
+    override val changesetComment = "Add seating info"
+    override val defaultDisabledMessage = R.string.default_disabled_msg_summer_outdoor_seating
+    override val wikiLink = "Key:outdoor_seating"
+    override val icon = R.drawable.ic_seating_quest
+    override val isReplaceShopEnabled = true
+
+    override val questTypeAchievements = listOf(QuestTypeAchievement.CITIZEN)
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_seating_name_title
+
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("""
+            nodes, ways with
+            amenity = bar
+            or amenity = pub
+            or amenity = food_court
+            or amenity = ice_cream
+            or amenity = restaurant
+            or amenity = cafe
+            or amenity = fast_food
```

Comment:
Why not use the same filter as above?

```suggestion
            nodes, ways with amenity ~ restaurant|cafe|fast_food|ice_cream|food_court|pub|bar
```